crates/rome_lsp/src/session.rs

```
@@ -119,7 +119,7 @@ impl Session {
         let lint_enabled = self.workspace.supports_feature(SupportsFeatureParams {
             feature: FeatureName::Lint,
             path: rome_path.clone(),
-        });
+        })?;
```

What's the reason that `supports_feature` now returns a `Result`?

---

If the workspace client is connected to a remote server then it's possible for the request to fail because it timed out or the connection was lost. Previously the error was hidden internally and the file was reported as unsupported instead, but that leads to unclear error messages for the user as supported files can be erroneously (and randomly) reported as unsupported by the CLI